README.md

```diff
@@ -161,6 +177,28 @@ This is not required, but pulling the image from the same region as your cluster
 - single node cluster degrades into unscheduleable on update (https://github.com/bottlerocket-os/bottlerocket/issues/501)
 - node labels are not automatically applied to allow scheduling (https://github.com/bottlerocket-os/bottlerocket/issues/504)
 
+## Troubleshooting
+
+When installed with the [suggested deployment](./update-operator.yaml), the logs can be fetched through Kubernetes deployment logs.
+To get logs run this with `kubectl` configured to the desired cluster namespace
+
+```sh
+kubectl logs deployment/update-operator-controller
+```
+
+Checking the logs is a good first step in understanding why something happened or didn't happen.
+
+### Why do only some of my Bottlerocket instances have an update available?
+
+Updates to Bottlerocket are rolled out in [waves](https://github.com/bottlerocket-os/bottlerocket/tree/develop/sources/updater/waves) to reduce the impact of issues; the container instances in your cluster may not all see updates at the same time.
+You can check whether an update is available on your instance by running the `apiclient update check` command from within the [control](https://github.com/bottlerocket-os/bottlerocket#control-container) or [admin](https://github.com/bottlerocket-os/bottlerocket#admin-container) container.
+
+### Why do new container instances launch with older Bottlerocket versions?
+
+The Bottlerocket update operator performs in-place updates for instances in your ECS cluster.
+The operator does not influence how those instances are launched.
+If you use an auto-scaling group to launch your instances, you can update the AMI ID in your launch configuration or launch template to use a newer version of Bottlerocket.
```

Comment:
Nit: if you are including links for how to setup kubectl, you may as well include links to the official AWS documentation about launch templates and auto scaling groups.

Reply:
Done.